lib/tasks/admins.rake

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+namespace :admins do
+  desc 'Update the list of admin users for the app'
+  task update: :environment do
+    if ENV["ORANGELIGHT_ADMIN_NETIDS"]
```

Comment:
Now that I'm looking more closely, do we even need to record the admin status in the database in the first place?  It seems like there is potential for the envvar and the database to get out of sync if folks forget to run this rake task.

Reply:
Yeah, I was trying to think about how to make sure they don't get out of sync as well. What if we added it as a monthly cron job? Part of what I like about adding it in the database, is that it makes it easy to set / unset your own admin status for testing.

Reply:
Can we accomplish the same with, say, `ORANGELIGHT_ADMIN_NETIDS='netid1 netid2' rails s` locally?

Or on a staging box, changing the app_config and reloading passenger?